python/foglamp/services/core/server.py

```diff
@@ -60,6 +60,34 @@ class Server:
     service_monitor = None
     """ foglamp.microservice_management.service_registry.Monitor """
 
+    _service_name = 'FogLAMP'
+    """ The name of this FogLAMP service """
+
+    _service_description = 'FogLAMP REST Services'
+    """ The description of this FogLAMP service """
+
+    _SERVICE_DEFAULT_CONFIG = {
+        'name': {
+            'description': 'The name of this FogLAMP service',
+            'type': 'string',
+            'default': 'FogLAMP'
+        },
+        'description': {
+            'description': 'The description of this FogLAMP service',
+            'type': 'string',
+            'default': 'The FogLAMP administrative API'
+        }
+    }
+
+    _MANAGEMENT_SERVICE = '_foglamp_manage._tcp'
```

Comment:
please remove `_` between foglamp and manage `'_foglampmanage._tcp'`

Reply:
No, this follows well known conventions.

Reply:
`_foglamp_manage` is standard instead of `_foglampmanage` you mean? I will look more then.